### PR TITLE
Fix admission metrics tests

### PR DESCRIPTION
**Pull request by tallclair**
Cleanup some typos in the admission metrics tests.

```release-note
NONE
```

### CODE REVIEW

**Review by jpbetz — APPROVED** [review]
lgtm

File: staging/src/k8s.io/apiserver/pkg/admission/metrics/metrics_test.go [hunk 1]
```diff
@@ -239,7 +239,7 @@ type mutatingFakeHandler struct {
 	admit bool
 }
 
-func (h *mutatingFakeHandler) Amit(a admission.Attributes) (err error) {
```
Comment (sttts) [inline staging/src/k8s.io/apiserver/pkg/admission/metrics/metrics_test.go]: That's why type assertions exist I guess.

**Pull request merged**